tests/test_init.py

```diff
@@ -53,3 +60,99 @@ def test_parse_bad_patchwork_url(self):
             "Can't parse patchwork url: '{}'".format(patchwork_url)
             in context.exception
         )
+
+
+class KBuilderTest(unittest.TestCase):
+    """Test cases for skt.kbuilder class"""
+    def setUp(self):
+        self.tmpdir = tempfile.mkdtemp()
+        self.tmpconfig = tempfile.NamedTemporaryFile()
+        self.kbuilder = skt.kbuilder(self.tmpdir, self.tmpconfig.name)
+        self.m_popen = Mock()
+        self.m_popen.returncode = 0
+        self.ctx_popen = mock.patch('subprocess.Popen',
+                                    Mock(return_value=self.m_popen))
+        self.ctx_check_call = mock.patch('subprocess.check_call', Mock())
+
+    def tearDown(self):
+        shutil.rmtree(self.tmpdir)
+
+    def test_clean_mktgz(self):
```

Comment:
Could you name those so that they all start with `test_mktgz_` and then have specifics on the end? E.g. `test_mktgz_clean`?